repos/fdbt-site/src/utils/index.ts

```diff
@@ -261,3 +263,50 @@ export const objectKeyMatchesExportNameExactly = (objectKey: string, exportName:
 export const fareTypeIsAllowedToAddACap = (fareType: string): boolean => {
     return !!fareType && ['single', 'return', 'flatFare'].includes(fareType);
 };
+
+export const getMyUniqueServices = (
+    services: { id?: string | number; lineId: string; startDate: string; endDate?: string }[],
+): string[] => {
+    const uniqueServiceList: string[] = [];
+    const uniqueServices: string[] = [];
+
+    services.forEach((service) => {
+        const serviceName = `${service.lineId}#${service.startDate}#${service.endDate ? service.endDate : ''}`;
+        if (!uniqueServiceList.includes(serviceName)) {
+            uniqueServices.push(String(service.id));
+            uniqueServiceList.push(serviceName);
+        }
+    });
+
+    return uniqueServices;
+};
+
+export const getUniqueServices = (services: ServiceType[]): ServiceType[] => {
+    const uniqueServiceList: string[] = [];
+    const uniqueServices: ServiceType[] = [];
+
+    services.forEach((service) => {
```

Comment:
I dont think it makes sense for us to have the same code 3 times over, slightly altered.